### ISSUE TITLE
Features Artist

### ISSUE THREAD

**Issue by screwfaze** [dkanada/gelli#21]
Hi,

I have noticed that featured artists from musicbrainz is showing on jellyfin but not on gelli not sure if anyone else has mentioned this.

Thanks

**Comment by DJWhitegold** [comment]
I noticed the same, but I prefer not to see the featuring artists. Perhaps gelli is pulling in album artist field. Whatever it's doing I actually prefer it

**Comment by screwfaze** [comment]
> I noticed the same, but I prefer not to see the featuring artists. Perhaps gelli is pulling in album artist field. Whatever it's doing I actually prefer it

I definitely prefer to see the featured artist so maybe can be an option?

**Comment by DJWhitegold** [comment]
> > I noticed the same, but I prefer not to see the featuring artists. Perhaps gelli is pulling in album artist field. Whatever it's doing I actually prefer it
> 
> I definitely prefer to see the featured artist so maybe can be an option?

Hopefully... good call.

**Comment by dkanada** [comment]
Do you guys just mean the regular `artist` field?

**Comment by screwfaze** [comment]
> Do you guys just mean the regular `artist` field?

![Screenshot_20200622-124501_Gelli](https://user-images.githubusercontent.com/17312955/85283997-42597e00-b486-11ea-833e-63bd9a3a54ab.jpg)
![Screenshot_20200622-124227_Jellyfin](https://user-images.githubusercontent.com/17312955/85284000-42f21480-b486-11ea-8103-91f4d253f72d.jpg)

See the example screenshots track one for the artists.

**Comment by Fabeuss** [comment]
Right, but if you start random play, there is no artist name like your screenshots... ([https://github.com/dkanada/gelli/issues/19])
